src/drivers/batt_smbus/parameters.c

```diff
@@ -49,3 +49,16 @@ PARAM_DEFINE_INT32(SENS_EN_BATT, 0);
  * @group Sensors
  */
 PARAM_DEFINE_FLOAT(BAT_C_MULT, 1.0f);
+
+/**
+ * Battery device model
+ *
+ * @reboot_required true
+ * @min 0
+ * @max 2
+ * @group Sensors
+ * @value 0 AutoDetect
+ * @value 1 BQ40Z50
+ * @value 2 BQ40Z80
+ */
+PARAM_DEFINE_INT32(BAT_MODEL, 0);
```

Comment:
`BAT_MODEL` is a bit too generic. Maybe something like `BAT_SMBUS_MODEL`? 

I'd also consider just skipping it entirely if the battery is easy to identify.

Reply:
I will be happy to make it auto-detect, and I am sure it possible, but didn't had time to do it.
any way, I only have the Z80 to check it with, so I will need someone with Z50 to check it too

Reply:
@dagar Done